dnf-plugin-system-upgrade.spec

```diff
@@ -1,6 +1,9 @@
+%{!?dnf_lowest_compatible: %global dnf_lowest_compatible 1.1.2}
+%{!?dnf_not_compatible: %global dnf_not_compatible 2.0}
```

Comment:
Where would dnf_lower_compatible and dnf_not_compatible be defined? I don't think there's a good reason not to unconditionally define those.